keys/src/address.rs

```diff
@@ -269,4 +309,71 @@ mod tests {
 		assert_eq!(address, "SVCbBs6FvPYxJrYoJc4TdCe47QNCgmTabv".into());
 		assert_eq!(address.to_string(), "SVCbBs6FvPYxJrYoJc4TdCe47QNCgmTabv".to_owned());
 	}
+
+	#[test]
+	fn test_from_to_cashaddress() {
+		let cashaddresses = vec![
+			"bitcoincash:qzxqqt9lh4feptf0mplnk58gnajfepzwcq9f2rxk55",
+			"bitcoincash:qr6m7j9njldwwzlg9v7v53unlr4jkmx6eylep8ekg2",
+			"bitcoincash:pq4ql3ph6738xuv2cycduvkpu4rdwqge5q2uxdfg6f",
+		];
+		let expected = vec![
```

Comment:
Could you please replace it with base58 encoded string representation? It will be more transparent then and easily validated by checking address in explorer, e.g. according to https://explorer.bitcoin.com/bch/address/bitcoincash:qzxqqt9lh4feptf0mplnk58gnajfepzwcq9f2rxk55 `qzxqqt9lh4feptf0mplnk58gnajfepzwcq9f2rxk55` corresponds to `1DmFp16U73RrVZtYUbo2Ectt8mAnYScpqM` so I think it should be shown in test.